Utilities/Test-VSProjectConsistency/Test-VSProjectConsistency.psm1

```diff
@@ -294,7 +295,7 @@ function Test-VSProjectConsistency
                 Where-Object { -not $PSItem.FullName.Substring($projectFolder.Length).StartsWith(".") } |
                 # If the file is inside a project folder then it's irrelevant for the current csproj.
                 Where-Object { -not (FileIsInsideAnyOfTheFolders $file.FullName $projectFoldersInTheProjectFolder) }
-                Where-Object { Test-Path $file.FullName -PathType Container } |
+            Where-Object { Test-Path $file.FullName -PathType Container } |
```

Comment:
The original indentation was good.

Reply:
According to the code it wasn't, but the code wasn't right either, because the previous line didn't have a pipe character at the end (so it probably wasn't even working correctly).